frontend/src/Dashboard/CrossingListPage/CrossingList.js

```diff
@@ -5,6 +5,8 @@ import crossingsQuery from './queries/crossingsQuery';
 import statusReasonsQuery from './queries/statusReasonsQuery';
 import statusDurationsQuery from './queries/statusDurationsQuery';
 import './CrossingList.css';
+import * as statusConstants from './CrossingListItem/StatusConstants';
```

Comment:
The way this gets imported made me curious about what was inside the Status Constants which led me to the file with come confusing commented out code. After talking to @briaguya, I updated the comment and committed to this branch. And created an issue #61 
https://github.com/cityofaustin/ctxfloods/commit/242f246c0670adb0d998ba114b01506a76cf2dc8#diff-2739d23e29ad5b9e31124c182d33047d